variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default     = "prod-"
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+
+variable "AUTH_CLIENT_ID" {
+  description = "GitHub client ID for authentication."
+}
+
+variable "AUTH_CLIENT_SECRET" {
+  description = "GitHub client secret for authentication."
```

Comment:
As with the outputs, we should [mark any sensitive input variables as such](https://learn.hashicorp.com/tutorials/terraform/sensitive-variables#reference-sensitive-variables) to avoid them potentially being printed out in logging anywhere